fmt/include\fmt/format-inl.h:1417:3: error: cannot use 'try' with exceptions disabled

Getting this error while compiling with LLVM clang++.exe on Windows, the exceptions are disabled `-fno-exceptions` and `-fms-compatibility` is not used: 
```
fmt/include\fmt/format-inl.h:1417:3: error: cannot use 'try' with exceptions disabled
 1417 |   FMT_TRY {
      |   ^
fmt/include\fmt\base.h:175:19: note: expanded from macro 'FMT_TRY'
  175 | #  define FMT_TRY try
      |  
```
Looks like relevant to #70 that does not seem to be properly/completely fixed.
The {fmt} version is [11.0.2](https://github.com/fmtlib/fmt/releases/tag/11.0.2).

The issue on 'master' persists:
```
fmt/include\fmt/format-inl.h:137:3: error: cannot use 'throw' with exceptions disabled
  137 |   throw format_error(message);
      |   ^
fmt/include\fmt/format-inl.h:1426:3: error: cannot use 'try' with exceptions disabled
 1426 |   FMT_TRY {
      |   ^
fmt/include\fmt\base.h:155:19: note: expanded from macro 'FMT_TRY'
  155 | #  define FMT_TRY try
      |                   ^
```

--- a/include/fmt/base.h
+++ b/include/fmt/base.h
@@ -146,6 +146,8 @@
 // Use the provided definition.
 #elif defined(__GNUC__) && !defined(__EXCEPTIONS)
 #  define FMT_USE_EXCEPTIONS 0
+#elif defined(__clang__) && !defined(__cpp_exceptions)
+#  define FMT_USE_EXCEPTIONS 0
 #elif FMT_MSC_VERSION && !_HAS_EXCEPTIONS
 #  define FMT_USE_EXCEPTIONS 0
 #else

